src/ui/server.ts

```diff
@@ -27,10 +27,10 @@ import * as suggestionController from './controllers/suggestion'
 import * as userController from './controllers/user'
 import * as xApiController from './controllers/xapi'
 
-/* tslint:disable:no-var-requires */
-const favicon = require('serve-favicon')
 /* tslint:disable:no-var-requires */
 const flash = require('connect-flash')
+const favicon = require('serve-favicon')
+/* tslint:enable */
```

Comment:
INFO - What happened to tslinting?

Reply:
Certain modules haven't been packaged to support `import` syntax so need to be required, which tslint rejects.